### ISSUE TITLE
It is possible to cheat with modifications to the image assets

### ISSUE THREAD

**Issue by NilanthAnimosus** [Regalis11/Barotrauma#260]
I know this doesn't sound serious, but I have noted you can also modify things like "Light" folder pngs, which can effect the lighting system and remove aspects like obstructing vision and proper LOS functionality so the XML system needs to begin checking for these in multiplayer somehow in the hash.

How this actually looks:
http://puu.sh/zs0sp/c74c2927e6.jpg
http://puu.sh/zs0y3/8b0ad02946.jpg

(And yes, this was multiplayer. it most definitely qualifies as a cheat).

**Comment by NilanthAnimosus** [comment]
I believe this is no longer applicable to the new lighting code, though it does not stop one from making crawlers extreme whites / easy to see colors and have things of interest stand out considerably.

**Comment by Regalis11** [comment]
There's not really a feasible way for us to prevent this, modified clients can change the lighting and other visuals even if we added some sort of hash checking. We could mitigate it by only sending information about dynamic entities (character & item position, etc) to clients that can see the entity, but in my opinion I don't see this as a big enough issue to warrant major changes to the netcode.

**Comment by NilanthAnimosus** [comment]
I understand a modified client can do it (L literally do that backwards for my own server mod even despite trying to keep it cheat-free all the same), but this is without any modifications. this is simply the image files being tweaked and is the only reason that I listed it. I talked to juan about it and found the lighting system has changed enough for this to cause no issues other than diving suit vision obscuring - so I don't feel its too serious either.
